omniledger/service/service.go

```diff
@@ -435,10 +434,64 @@ func (s *Service) updateCollectionCallback(sbID skipchain.SkipBlockID) error {
 		s.darcToScMut.Lock()
 		s.darcToSc[string(d.GetBaseID())] = sb.SkipChainID()
 		s.darcToScMut.Unlock()
+		// create the view-change manager entry
+		s.viewChangeMan.add(s.sendViewChangeReq, s.sendNewView, s.isLeader, string(sb.Hash))
```

Comment:
I kept them separate because we might need to do restarts later, when the timeout duration or threshold changes. Nevertheless, I have something in the `viewChangeManager` that says `start` should be called.